README.md

```diff
@@ -103,7 +103,8 @@ Run `python core.py validate --help` to see the list of validation options.
                                   files
   --meddra TEXT                   Path to directory with MedDRA dictionary
                                   files
-  -r, --rules TEXT                Rule core id. ex: CORE-000001. Can be specified multiple times.
+  -r, --rules TEXT                Specify rule core ID ex. CORE-000001. Can be specified multiple times.
+  -lr, --local_rules TEXT          Specify absolute path to unpublished rule yml and/or json rule file.
```

Comment:
There are still references to "unpublished" here and 2 other places. Should be replaced with "local"